### PR TITLE
chore(react-northstar): add missing project tags and add explicit bundle dep on v9 package from v0 app

### PR DESCRIPTION
**Pull request by Hotell**
## New Behavior

see PR title

## Related Issue(s)

- Unblocks https://github.com/microsoft/fluentui/pull/32168

### CODE REVIEW

File: packages/fluentui/state/project.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "name": "state",
   "$schema": "../../../node_modules/nx/schemas/project-schema.json",
   "projectType": "library",
-  "implicitDependencies": []
+  "implicitDependencies": [],
+  "tags": ["react-northstar"]
```
Comment (Hotell) [inline packages/fluentui/state/project.json:6]: these are used to create path aliases as part of webpack build https://github.com/microsoft/fluentui/blob/master/scripts/gulp/src/config.ts#L16-L30

File: packages/fluentui/docs/project.json [hunk 2]
```diff
@@ -10,6 +10,10 @@
         {
           "projects": ["react-northstar"],
           "target": "build:info"
+        },
+        {
+          "projects": ["react-portal-compat-context"],
```
Comment (Hotell) [inline packages/fluentui/docs/project.json:15]: docs uses webpack path aliases, but it also depends on v9 package which cannot be used as alias (https://github.com/microsoft/fluentui/blob/master/scripts/gulp/src/config.ts#L12), thus we orchestrate physical build of this cross framework package

**Review by dmytrokirpa — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
ButtonMinimalPerf.default | [103](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonMinimalPerf.default.html) | [88](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.17:1
FlexMinimalPerf.default | [165](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/FlexMinimalPerf.default.html) | [145](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.14:1
PortalMinimalPerf.default | [91](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/PortalMinimalPerf.default.html) | [82](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.11:1
TreeWith60ListItems.default | [92](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TreeWith60ListItems.default.html) | [83](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.11:1
IconMinimalPerf.default | [395](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/IconMinimalPerf.default.html) | [363](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.09:1
ListMinimalPerf.default | [322](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListMinimalPerf.default.html) | [299](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.08:1
AttachmentMinimalPerf.default | [85](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AttachmentMinimalPerf.default.html) | [81](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.05:1
BoxMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/BoxMinimalPerf.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.05:1
VideoMinimalPerf.default | [453](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/VideoMinimalPerf.default.html) | [432](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.05:1
ChatDuplicateMessagesPerf.default | [161](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.04:1
CheckboxMinimalPerf.default | [1150](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1108](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.04:1
DropdownManyItemsPerf.default | [400](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DropdownManyItemsPerf.default.html) | [386](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.04:1
GridMinimalPerf.default | [193](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/GridMinimalPerf.default.html) | [186](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.04:1
RefMinimalPerf.default | [113](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RefMinimalPerf.default.html) | [109](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.04:1
SkeletonMinimalPerf.default | [211](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SkeletonMinimalPerf.default.html) | [203](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.04:1
TextAreaMinimalPerf.default | [295](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TextAreaMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.04:1
ButtonOverridesMissPerf.default | [677](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [658](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.03:1
CardMinimalPerf.default | [317](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CardMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.03:1
CarouselMinimalPerf.default | [269](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CarouselMinimalPerf.default.html) | [261](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.03:1
ProviderMinimalPerf.default | [205](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ProviderMinimalPerf.default.html) | [199](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.03:1
TableMinimalPerf.default | [234](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TableMinimalPerf.default.html) | [227](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.03:1
TooltipMinimalPerf.default | [1291](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TooltipMinimalPerf.default.html) | [1259](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.03:1
TreeMinimalPerf.default | [485](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TreeMinimalPerf.default.html) | [472](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.03:1
AnimationMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AnimationMinimalPerf.default.html) | [292](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.02:1
ButtonSlotsPerf.default | [305](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonSlotsPerf.default.html) | [298](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.02:1
DropdownMinimalPerf.default | [1423](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DropdownMinimalPerf.default.html) | [1391](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.02:1
ProviderMergeThemesPerf.default | [663](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [650](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.02:1
RadioGroupMinimalPerf.default | [258](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [254](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.02:1
SegmentMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SegmentMinimalPerf.default.html) | [196](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.02:1
AlertMinimalPerf.default | [165](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AlertMinimalPerf.default.html) | [164](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [441](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatMinimalPerf.default.html) | [436](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [3585](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DatepickerMinimalPerf.default.html) | [3544](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.01:1
FormMinimalPerf.default | [231](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/FormMinimalPerf.default.html) | [229](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [725](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [717](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
ListWith60ListItems.default | [367](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListWith60ListItems.default.html) | [362](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.01:1
MenuButtonMinimalPerf.default | [956](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [950](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.01:1
PopupMinimalPerf.default | [345](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/PopupMinimalPerf.default.html) | [343](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.01:1
ReactionMinimalPerf.default | [212](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ReactionMinimalPerf.default.html) | [209](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.01:1
SliderMinimalPerf.default | [738](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SliderMinimalPerf.default.html) | [731](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
ToolbarMinimalPerf.default | [549](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ToolbarMinimalPerf.default.html) | [545](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.01:1
AttachmentSlotsPerf.default | [635](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AttachmentSlotsPerf.default.html) | [633](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1:1
ChatWithPopoverPerf.default | [204](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatWithPopoverPerf.default.html) | [204](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1:1
DialogMinimalPerf.default | [432](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DialogMinimalPerf.default.html) | [434](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1:1
ImageMinimalPerf.default | [227](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ImageMinimalPerf.default.html) | [226](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1:1
InputMinimalPerf.default | [538](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/InputMinimalPerf.default.html) | [537](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/InputMinimalPerf.default_base.html) | 1:1
LayoutMinimalPerf.default | [201](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LayoutMinimalPerf.default.html) | [200](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1:1
TableManyItemsPerf.default | [1086](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TableManyItemsPerf.default.html) | [1090](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [1471](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CustomToolbarPrototype.default.html) | [1469](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
AvatarMinimalPerf.default | [110](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AvatarMinimalPerf.default.html) | [111](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 0.99:1
EmbedMinimalPerf.default | [1892](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/EmbedMinimalPerf.default.html) | [1905](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.99:1
HeaderMinimalPerf.default | [208](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/HeaderMinimalPerf.default.html) | [211](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 0.99:1
HeaderSlotsPerf.default | [463](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/HeaderSlotsPerf.default.html) | [470](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 0.99:1
ListCommonPerf.default | [395](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListCommonPerf.default.html) | [401](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListCommonPerf.default_base.html) | 0.99:1
AccordionMinimalPerf.default | [84](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AccordionMinimalPerf.default.html) | [86](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.98:1
DividerMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DividerMinimalPerf.default.html) | [204](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/DividerMinimalPerf.default_base.html) | 0.98:1
LoaderMinimalPerf.default | [188](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LoaderMinimalPerf.default.html) | [192](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 0.98:1
RosterPerf.default | [1606](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RosterPerf.default.html) | [1641](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/RosterPerf.default_base.html) | 0.98:1
SplitButtonMinimalPerf.default | [2293](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2329](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 0.98:1
TextMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TextMinimalPerf.default.html) | [202](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.98:1
ListNestedPerf.default | [323](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListNestedPerf.default.html) | [334](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/ListNestedPerf.default_base.html) | 0.97:1
MenuMinimalPerf.default | [493](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/MenuMinimalPerf.default.html) | [506](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.97:1
StatusMinimalPerf.default | [390](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/StatusMinimalPerf.default.html) | [404](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/StatusMinimalPerf.default_base.html) | 0.97:1
LabelMinimalPerf.default | [216](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LabelMinimalPerf.default.html) | [227](https://fluentuipr.z22.web.core.windows.net/pull/32210/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.95:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Pull request merged**